src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
We should move this to `StartImportingBasicBlock` - everything related to stack management is in the ILToCppCompiler, so this little piece should be there as well.

I would move the `_stack` field to ILToCppCompiler as well.